app/resolve/test/test_return.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+from core.query_solvers import AvailabilityQuery, DistributionQuery, solve_availability
+from core.db_manager import SyncDBManager
+from core.rquest_dto import cohort
+from core.rquest_dto.result import RquestResult
+from core.rquest_dto.cohort import Cohort
+from core.rquest_dto.group import Group
+from core.rquest_dto.rule import Rule
+from dotenv import load_dotenv
+import os
+import core.config as config
+
+load_dotenv()
+
+
+@pytest.fixture
```

Comment:
Tests! Wonderful.
Let's do an issue to run these in CI, and make them easier to run locally too (for example vscode integration is nice)